lib/events-scraper/event.rb

```diff
@@ -65,7 +65,7 @@ def remote_document
       uri = "#{WEBSITE}#{@link}?do=export_xhtmlbody"
       print "\n #{@index+1} ".yellow
       print "#{WEBSITE}#{@link}.html ".blue
-      Nokogiri::HTML(open(uri))
+      Nokogiri::HTML(open(uri), nil,"UTF-8")
```

Comment:
the last argument wants some space to breathe.  Pacman!

(I'm wondering if `UTF-8` is not the default anyway?)